fuzzing/config/config.go

```diff
@@ -113,6 +113,9 @@ type AssertionTestingConfig struct {
 
 	// TestViewMethods dictates whether constant/pure/view methods should be tested.
 	TestViewMethods bool `json:"testViewMethods"`
+
+	// AssertOptions describes options for testing panics using the panic codes.
+	AssertOptions []byte `json:"assertOptions"`
```

Comment:
So after some thought I don't think we want a `[]byte` for this. The reason being it sort of diminishes the user experience since the user will have to go research what the different panic codes are and what their hex representations are. The better way would be to create an object that can individually enable each panic code.

e.g.

```
type AssertionTestingConfig struct {
...
AssertionModes AssertionModesConfig `json:"assertionModes"`
...
}

type AssertionModesConfig struct {
	FailOnArithmeticUnderflow bool `json:"failOnArithmeticUnderflow"`
}
```

This way the user can just set things to true and false. In config_defaults, we can have `FailOnAssertion` as the only one as `true` and the rest can be false.

Reply:
+1 to what Anish said. As a note, I'd just re-reference the issue associated to this, as it provided a list to medusa panic code definitions to leverage and mentioned designing this with bools instead. It might provide you more useful context here. We'd want one config option (bool) per panic code in medusa in the end.